appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 4.0.0.{build}
+platform:
+    - x64
+install:
+    - SET PATH=C:\Ruby215\bin;%PATH%
+    - gem install bundler --quiet --no-ri --no-rdoc
+    - bundle install --without development
+build: off
+test_script:
+    - bundle exec rspec spec
```

Comment:
I use if, I'm just curious if this is something that folks would be okay with. 

Here's how I set it up for choco - https://github.com/chocolatey/choco/blob/master/appveyor.yml#L36-L43